api.planx.uk/send/uniform.ts

```diff
@@ -225,7 +225,10 @@ async function authenticate({
     throw Error("Failed to authenticate to Uniform - no access token returned");
   }
 
-  if (!response.data["organisation-name"] || response.data["organisation-id"]) {
+  if (
+    !response.data["organisation-name"] ||
+    !response.data["organisation-id"]
+  ) {
```

Comment:
The missing `!` here is my schoolboy error here that caused the latest Uniform submission to fail sorry!

If I can't progress this PR by later today I'll cherry-pick this commit into a new PR to get to prod for re-submission to take place.